test/gtest/uct/test_p2p_err.cc

```diff
@@ -231,6 +231,10 @@ UCS_TEST_P(uct_p2p_err_test, invalid_am_short_length) {
 
 UCS_TEST_P(uct_p2p_err_test, invalid_am_bcopy_length) {
     check_caps(UCT_IFACE_FLAG_AM_BCOPY | UCT_IFACE_FLAG_ERRHANDLE_BCOPY_LEN);
+    if (RUNNING_ON_VALGRIND) {
```

Comment:
@yosefe Because we need other tests in uct_p2p_err_test suite. This test (bcopy) will failed under valgrind because we can't check message length before memcpy called in callback (pack_cb).